gstd/gstd_pipeline.c

```diff
@@ -268,6 +296,14 @@ gstd_pipeline_dispose (GObject * object)
     self->elements = NULL;
   }
 
+  if (self->position) {
```

Comment:
No need to ask here. The checks above are because those are pointers.

Reply:
Done